CHANGELOG.md

```diff
@@ -8,6 +8,60 @@ The format is based on [these versioning and changelog guidelines](https://git.i
 
 ---
 
+## 4.9.0 - 2019-12-04
+
+### Enhancements
+
+- Added `external` prop to `ResourceList` ([#2408](https://github.com/Shopify/polaris-react/pull/2408))
+- Added `onMouseEnter` and `onTouchStart` props to `Button` ([#2409](https://github.com/Shopify/polaris-react/pull/2409))
+- Added `ariaHaspopup` prop to `Popover` ([#2248](https://github.com/Shopify/polaris-react/pull/2248))
+- Fixed an accessibility issue where the `Form` implicit submit was still accessible via keyboard ([#2447](https://github.com/Shopify/polaris-react/pull/2447))
+- Moved `Button` styles from the `Buttongroup` CSS file to the `Button` CSS file ([#2441](https://github.com/Shopify/polaris-react/pull/2441))
```

Comment:
Should this be in the code quality section?

Reply:
I wouldn't think so, but then again I put it here :)

Reply:
I guess the question is: from a Polaris React consumer's perspective, would this be an enhancement, or does it fall under the "something was changed under the hood" umbrella?

Reply:
I would say enhancement because it makes `ButtonGroups` more flexible allowing to remove custom solutions that were implemented where the current implementation didn't work.

Reply:
I'm not sure this context is clear from reading the changelog item, but that makes sense.